tests/Middleware/NonAvailablePlaceHTMLFormatterTest.php

```diff
@@ -112,4 +113,34 @@ function () {
 
         $this->assertEquals('foo', $result);
     }
+
+    public function testItWillAddTitleAttributeWithReason(): void
+    {
+        $place = new Place();
+        $place->setStatus(new Status('Unavailable', new TranslatedString(['nl' => 'Covid-19'])));
+
+        $result = $this->formatter->format(
+            $place,
+            function () {
+                return 'foo';
+            }
+        );
+
+        $this->assertEquals('<span title="Covid-19" class="cf-meta">Permanent gesloten</span>', $result);
+    }
+
+    public function testItWillNotAddTitleAttributeWhenReasonIsNotAvailableInCorrectLanguage(): void
```

Comment:
Ah, I was under the impression that we should use the reason from the `mainLanguage` is there was no for the selected language. But it could be either way. Did you check with @corneelwille by any chance? Or was it mentioned in the ticket?

Reply:
Good point, I already forgot about that. Lemme check quickly.

Reply:
Corneel confirmed that it's better not to show anything right now. We can always change it later if that decision changes.